vispy/visuals/regular_polygon.py

```diff
@@ -20,56 +19,41 @@ class RegularPolygonVisual(EllipseVisual):
     Parameters
     ----------
 
-    pos : array
+    center : array-like (x, y)
         Center of the regular polygon
     color : str | tuple | list of colors
         Fill color of the polygon
     border_color : str | tuple | list of colors
         Border color of the polygon
+    border_width: float
+        The width of the border in pixels
     radius : float
         Radius of the regular polygon
         Defaults to  0.1
     sides : int
         Number of sides of the regular polygon
     """
-    def __init__(self, pos=None, color='black', border_color=None,
-                 radius=0.1, sides=4, **kwargs):
-        super(EllipseVisual, self).__init__()
-        self.mesh.mode = 'triangle_fan'
-        self._pos = pos
-        self._color = Color(color)
-        self._border_color = Color(border_color)
-        self._radius = radius
-        self._sides = sides
-        self._update()
+    def __init__(self, center=None, color='black', border_color=None,
+                 border_width=1, radius=0.1, sides=4, **kwargs):
```

Comment:
Can the inheritance be changes so that we have RegularPolygon and EllipseVisual inherit from that? That makes much more sense semantically... (but in a new PR, not this one)